### PR TITLE
All new header and sticky footer

### PR DESCRIPTION
**Pull request by naveenbhaskar**

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/_navigation.scss [hunk 1]
```diff
@@ -14,100 +22,72 @@
...
-
-  .is-dropdown-submenu {
-    border: none;
-  }
-
-  .user {
-    @include icon-before($type: user, $margin: auto);
-  }
-}
+//.navigation {
```
Comment (ketan) [inline server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/_navigation.scss]: Remove the commented out code please?

Comment (naveenbhaskar) [inline server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/_navigation.scss]: yes.. removed and pushed.

**Comment by ketan** [comment]
@naveenbhaskar LGTM, could you check the login page footer too (http://localhost:8153/go/auth/login)

File: server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/_footer.scss [hunk 2]
```diff
@@ -13,74 +13,79 @@
...
+  font-size  : 1.2rem;
+  padding    : 1rem;
 
   .inline-list a {
-    display:       inline-block;
-    border-radius: 2px;
+    display       : inline-block;
+    border-radius : 2px;
 
     &:nth-of-type(3) {
```
Comment (ketan) [inline server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/_footer.scss]: This selector was added for the green separator bar, I don't suppose we have the bar, is there any reason to keep the selector with a padding?

**Comment by ketan** [comment]
I checked the agent SPA, and it looks weird now —

screen shot 2016-08-29 at 1 11 16 pm


/cc @iamsumanth

**Comment by naveenbhaskar** [comment]
ok . Will fix it

**Pull request merged**